### DISCUSSION TITLE
posts/blog-home-server-2

### DISCUSSION THREAD

**Discussion by giscus** [witch-factory/witch-next-blog#14] [General]
# posts/blog-home-server-2

블로그를 홈 서버에 올려보자

https://blog.witch.work/posts/blog-home-server-2

**Comment by binary-ho** [comment]
진짜 미쳤네요

**Comment by Kim-Hyunjo** [comment]
OMG

**Comment by hux2** [comment]
안녕하세요. 글 너무 잘 봤습니다. 저도 비슷하게 Proxmox에 OPNsense를 설치해서 방화벽/라우터로 쓰고 있는데요, 혹시 글쓴이님은 PfSense VM을 재부팅하면 WAN ip를 못받아오는 문제는 없으신가요?? 저는 이 문제 때문에 골치가 아프네요.. ㅠㅠ

**Reply by witch-factory** [reply]
안녕하세요. 글 읽어 주셔서 감사합니다. 말씀하신 부분을 정확히는 알 수 없지만 IP 설정 등에서 뭔가 문제가 생긴 게 아닐까요? 제가 겪은 문제가 아니라서 도움을 드릴 수가 없어서 죄송합니다..

**Reply by hux2** [reply]
기존에 물리 NIC 하나를 OPNsense VM에 패스스루하여 구성했었는데, 혹시나 싶어 Linux Bridge로 묶어서 넘겨줬더니 수많은 재부팅 후에도 공인 IP를 문제없이 받아오네요.
OPNsense와 Realtek 랜 사이의 호환 문제였는지, Proxmox에서 패스스루를 해준게 뭔가 꼬였는지 잘 모르겠지만 가상화하여 넘겨주니 잘 되어 다행입니다.
작성하신 글을 보며 세팅에 많은 도움을 받았습니다. 고맙습니다!

**Comment by binary-ho** [comment]
아나 실화냐? ㅋㅋㅋ
당근에서 데탑 하나 사고 검색했더니 니 글 뜨네..
이거 먼저 봤어야했는데 하아

**Reply by witch-factory** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ근데 데탑이 뭐 딴데 쓸수도 있고 메모리 늘리기도 좋고 장점도 있을 듯
